docs/source/user_guide/supported_features.md

```diff
@@ -0,0 +1,20 @@
+# Supported Features
+
+| Feature                       | V0 | V1 |
+|-------------------------------|----|----|
+| Quantization                  |    |✅︎  |
+| LoRA                          |⚠️  |    |
```

Comment:
🤔 

Well I haven't tried enabling lora but I also don't see anything in FMS about it being supported so I'd be kinda surprised if it worked